### PR TITLE
enh(UI): Take downtime options default value in RS

### PR DESCRIPTION
**Pull request by Jordan91**
## Description

Take downtime options default value in resource status

https://www.loom.com/share/ef583fcabaf14f369306f380d3e5bdc0?from_recorder=1&focus_title=1

## Type of change

- [x] New functionality (non-breaking change)

## How to reproduce

Go on Administration  >  Parameters  >  Monitoring  and set or unset checkbox for downtime. Then Go to resource status to check if when you want  to set downtime default option are the same as the general parameters.

## Target serie

- [x] 21.10
- [x] 22.04.x (master)

<h2> How this pull request can be tested ? </h2>

Go to `Administration  >  Parameters  >  Monitoring` and check or uncheck option default for downtime:
- Fixed
- Set downtimes on services attached to hosts 

To finish  save your changes.
Then go to Resource Status, and set a downtime by passing in  the colum status  and try again by click on the Big Chip "Set Downtime" above the Listing.

### CODE REVIEW

**Review by Thebarda — DISMISSED** [review]

File: doc/API/centreon-api-v22.04.yaml [hunk 1]
```diff
@@ -3401,6 +3401,14 @@ paths:
                       type: boolean
                       description: Default monitoring acknowledgement sticky option
                       example: true
+                      monitoring_default_downtime_fixed:
```
Comment (kduret) [inline doc/API/centreon-api-v22.04.yaml:3404]:
```suggestion
                    monitoring_default_downtime_fixed:
```

File: www/front_src/src/Resources/Details/tabs/Details/CommandLine/index.tsx [hunk 2]
```diff
@@ -14,9 +14,9 @@ const useStyles = makeStyles((theme) => ({
     display: 'flex',
     marginLeft: theme.spacing(1),
   },
-  command: {
-    fontWeight: theme.typography.fontWeightBold,
-  },
+  // command: {
+  //   fontWeight: theme.typography.fontWeightBold,
+  // },
```
Comment (kduret) [inline www/front_src/src/Resources/Details/tabs/Details/CommandLine/index.tsx:19]: remove comments

File: doc/API/centreon-api-v22.04.yaml (hunk 1 above)
Comment (Jordan91) [reply doc/API/centreon-api-v22.04.yaml]: Thank's Kevin

File: www/front_src/src/Resources/Details/tabs/Details/CommandLine/index.tsx [hunk 3]
```diff
@@ -46,9 +46,7 @@ const CommandWithArguments = ({ commandLine }: Props): JSX.Element => {
                   |
                 </Typography>
               )}
-              <Typography className={classes.command} variant="body2">
-                {command}
-              </Typography>
+              <Typography variant="body2">{command}</Typography>
```
Comment (Jordan91) [inline www/front_src/src/Resources/Details/tabs/Details/CommandLine/index.tsx:49]: missing className for style
=> `className={classes.command}`

File: src/Centreon/Application/Controller/Administration/ParametersController.php [hunk 4]
```diff
@@ -106,6 +114,12 @@ public function getParameters(): View
                 case self::DEFAULT_ACKNOWLEDGEMENT_STICKY:
                     $isAcknowledgementSticky = (int) $option->getValue() === 1 ? true : false;
                     break;
+                case self::DEFAULT_DOWNTIME_WITH_SERVICES:
+                    $isDowntimeWithServices = (int) $option->getValue() === 1 ? true : false;
+                    break;
+                case self::DEFAULT_DOWNTIME_FIXED:
+                    $isDowntimeFixed = (int) $option->getValue() === 1 ? true : false;
+                     break;
```
Comment (kduret) [inline src/Centreon/Application/Controller/Administration/ParametersController.php:122]:
```suggestion
                    break;
```

**Review by kduret — DISMISSED** [review]
ok for BE

File: www/front_src/src/Resources/Details/tabs/Details/CommandLine/index.tsx [hunk 5]
```diff
@@ -15,7 +15,7 @@ const useStyles = makeStyles((theme) => ({
     marginLeft: theme.spacing(1),
   },
   command: {
-    fontWeight: theme.typography.fontWeightBold,
+    fontWeight: 'bold',
```
Comment (Thebarda) [inline www/front_src/src/Resources/Details/tabs/Details/CommandLine/index.tsx:18]:
```suggestion
    fontWeight: theme.typography.fontWeightBold,
```

File: www/front_src/src/Resources/Filter/Criterias/index.tsx [hunk 6]
```diff
@@ -31,15 +33,36 @@ const useStyles = makeStyles((theme) => ({
...
+  const criterias = sortBy<Criteria>(
+    ({ name }) => criteriaNameSortOrder[name],
+    filterWithParsedSearch.criterias,
+  );
+
+  return pipe(
+    reject(isNonSelectableCriteria) as (criterias) => Array<Criteria>,
+  )(criterias);
+};
+
```
Comment (Thebarda) [inline www/front_src/src/Resources/Filter/Criterias/index.tsx:52]: I guess you need to wait for the other PR

File: www/front_src/src/Resources/Details/tabs/Details/CommandLine/index.tsx (hunk 5 above)
Comment (Jordan91) [reply www/front_src/src/Resources/Details/tabs/Details/CommandLine/index.tsx:18]: I get  typing issue if i set theme.typo.... He don't accept string | undefined

File: www/front_src/src/Resources/Filter/Criterias/index.tsx (hunk 6 above)
Comment (Jordan91) [reply www/front_src/src/Resources/Filter/Criterias/index.tsx:52]: ok

File: www/front_src/src/Resources/Details/tabs/Details/CommandLine/index.tsx (hunk 5 above)
Comment (Jordan91) [reply www/front_src/src/Resources/Details/tabs/Details/CommandLine/index.tsx:18]: As BDA said , there is a typing problem, in waiting to resolve issue , i fixe like that temporaly

**Review by Thebarda — APPROVED** [review]

**Review by jdelpierre — APPROVED** [review]

**Review by kduret — APPROVED** [review]

**Pull request merged**